examples/voltage_in_ratio.rs

```diff
@@ -0,0 +1,140 @@
+// phidget-rs/examples/voltage_in.rs
+//
+// Copyright (c) 2023, Frank Pagliughi
+// implemented by Jorge Guerra and Riley Hernandez 2024
```

Comment:
You are welcome to claim copyright for the code you added. As this is copypasta from a file I created, it would be proper to add your copyright line below mine, like:
```
// Copyright (c) 2023, Frank Pagliughi
// Copyright (c) 20234 Jorge Guerra and Riley Hernandez
```